src/useKeyboardEvent/useKeyboardEvent.stories.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable unicorn/prevent-abbreviations,import/no-extraneous-dependencies */
+import { bind, computed, defineComponent, reactive } from '@muban/muban';
+import type { Story } from '@muban/storybook/types-6-0';
+import { html } from '@muban/template';
+import { Key } from 'ts-key-enum';
+import { useKeyboardEvent } from './useKeyboardEvent';
+
+export default {
+  title: 'useKeyboardEvent',
+};
+
+export const Demo: Story = () => ({
+  component: defineComponent({
+    name: 'story',
+    refs: {
+      label: 'label',
+      inputField: 'input-field',
+    },
+    setup({ refs }) {
+      const state = reactive<Array<string>>([]);
+
+      const log = (message: string) => {
```

Comment:
Should we make an (internal) hook for these kind of story helpers? :)

Reply:
I've created a new PR (https://github.com/mubanjs/muban-hooks/pull/15) that adds the hook, since we want this implemented on all the open PR's :)